src/celestia/celx_celestia.cpp

```diff
@@ -2475,6 +2475,28 @@ static int celestia_getrootcategories(lua_State *l)
     return celx.pushIterable<UserCategory*>(set);
 }
 
+static int celestia_bindtextdomain(lua_State *l)
+{
+    CelxLua celx(l);
+
+    const char *domain = celx.safeGetString(2, AllErrors, "First argument of celestia:bindtranslationdomain must be domain name string.");
+    if (domain == nullptr)
```

Comment:
I suppose `domain == nullptr` is not enough, you should also check that domain != ""
`if (domain == nullptr || *domain == '\0')`